Source/Microsoft.Gateway/Microsoft.InnerEye.DicomConstraints/Implementation/BaseOrderConstraint.cs

```diff
@@ -9,44 +9,48 @@ namespace Microsoft.InnerEye.DicomConstraints
     /// <summary>
     /// static class to compute a constraint value
     /// </summary>
-    /// <typeparam name="TCompare"></typeparam>
-    /// <typeparam name="TExtract"></typeparam>
-    /// <typeparam name="TSelector"></typeparam>
+    /// <typeparam name="TCompare">Instance of IComparable.</typeparam>
+    /// <typeparam name="TExtract">Type of value extracted from DICOM tag.</typeparam>
+    /// <typeparam name="TSelector">Instance of ISelector.</typeparam>
     internal static class BaseOrderConstraint<TCompare, TExtract, TSelector>
         where TCompare : IComparable, new()
         where TSelector : ISelector<TExtract, TCompare>, new()
     {
         /// <summary>
         /// Runs the ordering constraint
         /// </summary>
-        /// <param name="dataSet"></param>
-        /// <param name="order"></param>
-        /// <param name="v"></param>
-        /// <param name="t"></param>
-        /// <param name="constraint"></param>
-        /// <param name="ordinal"></param>
-        /// <returns></returns>
+        /// <param name="dataSet">DICOM dataset to test.</param>
+        /// <param name="order">Order between value in DICOM tag and v.</param>
+        /// <param name="v">Value to test.</param>
+        /// <param name="t">DICOM tag.</param>
+        /// <param name="constraint">Source constraint.</param>
+        /// <param name="ordinal">Tag ordinal.</param>
+        /// <returns>New DicomConstraintResult.</returns>
         public static DicomConstraintResult Check(DicomDataset dataSet, Order order, TCompare v, DicomTag t, DicomConstraint constraint, int ordinal = 0)
```

Comment:
expanding the names of this parameters might be helpful

Reply:
Done